src/main/java/uk/ac/ic/wlgitbridge/server/GitBridgeServer.java

```diff
@@ -57,11 +64,36 @@ public GitBridgeServer(
                 rootGitDirectoryPath,
                 config.getRepoStore().flatMap(RepoStoreConfig::getMaxFileSize)
         );
-        DBStore dbStore = new SqliteDBStore(
-                Paths.get(
-                        repoStore.getRootDirectory().getAbsolutePath()
-                ).resolve(".wlgb").resolve("wlgb.db").toFile()
-        );
+
+        DBStore dbStore;
+        // TODO: move this to a fromConfig function
+        Optional<DatabaseConfig> maybeConfig = config.getDatabase();
+        if (maybeConfig.isPresent()) {
+          DatabaseConfig databaseConfig = maybeConfig.get();
+          if (databaseConfig.getDatabaseType() == DatabaseConfig.DatabaseType.Postgres) {
+            Log.info("Database: connect to postgres");
+            PostgresOptions options = ((PostgresConfig)databaseConfig).getOptions();
+            dbStore = new PostgresDBStore(
+              options.getUrl(),
+              options.getUsername(),
+              options.getPassword()
+            );
+          } else {
+            Log.info("Database: connect to sqlite");
```

Comment:
Looks like this code is duplicated in the next `else` statement as well. Can this be simplified?